python/packages/autogen-agentchat/src/autogen_agentchat/messages.py

```diff
@@ -23,6 +23,9 @@ class BaseMessage(BaseModel, ABC):
     models_usage: RequestUsage | None = None
     """The model client usage incurred when producing this message."""
 
+    metadata: Dict[str, str] | None = None
```

Comment:
@ekzhu .. do you see any side effects here?
This type of field seems reasonable

Reply:
We need to be more specific on the fields rather than a kitchen sink.